assets/js/state/catalog.js

```diff
@@ -1,5 +1,12 @@
 import { createSlice } from '@reduxjs/toolkit';
 
+export const UPDATE_CATALOG = 'UPDATE_CATALOG';
+
+export const updateCatalog = (env = {}) => ({
```

Comment:
Could redo them using redux `createAction` util?
We already use it in many other places, and it is the recommended way of creating actions

Reply:
Thanks for the suggestion, i used the createAction helper :+1: